javarosa/src/main/java/org/javarosa/xpath/expr/XPathFormatDateForCalendarFunc.java

```diff
@@ -47,4 +47,9 @@ private static String formatDateForCalendar(Object dateObject, Object format) {
             throw new XPathUnsupportedException("Unsupported calendar type: " + format);
         }
     }
+
+    @Override
+    public String getDocumentation() {
+        return "?";
```

Comment:
@wpride I had trouble finding examples of this function. Can you fill out the documentation here and update [CommCare Functions confluence page](https://confluence.dimagi.com/display/commcarepublic/CommCare+Functions)?